### DISCUSSION TITLE
point_0723

### DISCUSSION THREAD

**Discussion by andrea29-star** [cloudsen12/dataset#334] [Roy]
"20200426T033529_20200426T034941_T47PPR"
![image](https://user-images.githubusercontent.com/54947195/147798766-69b237d2-f064-4989-82bb-7bd6348269c0.png)

"20200302T033641_20200302T034901_T47PPR"
![image](https://user-images.githubusercontent.com/54947195/147798838-2f842e88-2293-4184-9c61-b23b9ef75f52.png)

"20191009T033639_20191009T034925_T47PPR"
![image](https://user-images.githubusercontent.com/54947195/147798895-57f48d99-6682-4e3a-b57e-345a6f0eda8e.png)

@ryali93

**Comment by AntoniCesar** [comment]
"20200426T033529_20200426T034941_T47PPR"
![Captura](https://user-images.githubusercontent.com/86502862/153644139-93e1673d-a1d2-46c3-8de5-dd432f73e577.JPG)

**Comment by AntoniCesar** [comment]
"20200302T033641_20200302T034901_T47PPR"
![Captura](https://user-images.githubusercontent.com/86502862/153649551-677b000b-e9ad-43f5-975e-db1c57974d1e.JPG)
